pkg/cli/build/build.go

```diff
@@ -199,6 +204,36 @@ func isImageDefinitionValid(ctx *image.Context) bool {
 	return true
 }
 
+func downloadKubernetesSELinuxPolicies(ctx *image.Context) error {
+	if ctx.ImageDefinition.Kubernetes.Version == "" {
+		return nil
+	}
+
+	configPath := filepath.Join(ctx.ImageConfigDir, "kubernetes", "config", "server.yaml")
```

Comment:
This module already imports from `combustion`. Is there a reason you can't have a method in the kubernetes file to return this path? Feels cleaner than having the specific config directory structure leak out into this file.

Reply:
I definitely see this as an issue but there's just no clean way to do it right now.. The `combustion` package understands the `/kubernetes/config` structure but it's the `kubernetes` package which knows that there's a `server.yaml` in there. I can extract it as suggested to somewhat improve it but the implementation will still bleed into the different packages in a way I don't like at all.